packages/zk-eddsa-event-ticket-pcd/src/ZKEdDSAEventTicketPCD.ts

```diff
@@ -294,6 +299,20 @@ function snarkInputForProof(
     revealTicketIsConsumed: fieldsToReveal.revealIsConsumed ? "1" : "0",
     ticketIsRevoked: ticketAsBigIntArray[7].toString(),
     revealTicketIsRevoked: fieldsToReveal.revealIsRevoked ? "1" : "0",
+    ticketCategory: ticketAsBigIntArray[8].toString(),
+    revealTicketCategory: fieldsToReveal.revealTicketCategory ? "1" : "0",
+    reservedSignedField1: ticketAsBigIntArray[9].toString(),
```

Comment:
These could all be hard-coded to 0 at this level.  I think we expect all of the TS code to need to be updated before these fields could be used.

To specifically apply this suggestion to other cases below as well, I think the reserved fields should always be set to 0 when passing data down to the snark, and always checked against -1 (throwing an error if they're not) on the output of the snark.